libraries/datahandler/src/main/thrift/licenses.thrift

```diff
@@ -123,6 +131,7 @@ struct License {
 
     20: optional list<Obligation> obligations,
     21: optional set<string> obligationDatabaseIds,
+    22: optional string obligationListId,
```

Comment:
Maybe it would have been better to completely remove `obligationDatabaseIds` and keep only `obligationListId`

Reply:
The main usage of obligationDatabaseIds, being in a set format, is for the bulk retrieval and setting of Obligations. If we were to replace it, modifications such as iterating over obligations and then fetching each id individually would likely be necessary.

Reply:
make sense